custom_components/frigate/sensor.py

```diff
@@ -96,6 +98,9 @@ def device_info(self) -> DeviceInfo:
             "identifiers": {get_frigate_device_identifier(self._config_entry)},
             "name": NAME,
             "model": self._get_model(),
+            "configuration_url": self._config_entry.data.get(
+                CONF_URL, self._config_entry.data.get(CONF_HOST, "")
```

Comment:
Ditto.

Reply:
Should I change this style of line to something like:
```python
"configuration_url": f"{self._config_entry.data[CONF_URL]}"
```
then?

Reply:
When there's no string manipulation required (like in this case), you can just do:

```py
"configuration_url": self._config_entry.data.get(CONF_URL)
```

(It's the CONF_HOST issue I'm referring to here).